docs/02.API-REFERENCE.md

```diff
@@ -969,12 +969,12 @@ Perform JavaScript `eval`.
 jerry_value_t
 jerry_eval (const jerry_char_t *source_p,
             size_t source_size,
-            bool is_strict);
+            uint32_t parse_opts);
```

Comment:
If this is an api change please add it to #2213.

Reply:
Thanks I've added it.